src/state/get-droppable-over.js

```diff
@@ -92,7 +101,8 @@ const getClippedAreaWithPlaceholder = ({
   }
 
   const isClippedByFrame: boolean = subject[droppable.axis.size] !== frame[droppable.axis.size];
-  const subjectWithGrowth = getWithGrowth(subject, requiredGrowth);
+
+  const subjectWithGrowth = getWithGrowth(clipped, requiredGrowth);
```

Comment:
this line is the fix. Rather than adding the growth to the initial subject, we add it to the updated clipped area